phpmyfaq/src/phpMyFAQ/Notification.php

```diff
@@ -76,10 +80,17 @@ public function sendNewFaqAdded(array $emails, int $faqId, string $faqLanguage):
                 $this->mail->addCc($email);
             }
             $this->mail->subject = $this->config->getTitle() . ': New FAQ was added.';
+            $this->faq->getRecord($faqId, null, true);
+            $link = $this->config->getDefaultUrl() . 'admin/?action=editentry&id=' . $faqId . '&lang=' . $faqLanguage;
             $this->mail->message = html_entity_decode(
                 Translation::get('msgMailCheck')
-            ) . "\n\n" . $this->config->getTitle() . ': ' . $this->config->getDefaultUrl(
-            ) . 'admin/?action=editentry&id=' . $faqId . '&lang=' . $faqLanguage;
+            ) . "<p><strong>Frage:</strong> " . $this->faq->getRecordTitle($faqId) . "</p>"
```

Comment:
A translation is missing for "Frage", you can use `Translation::get('msgAskYourQuestion')`